test/units/TEST-72-SYSUPDATE.sh

```diff
@@ -5,6 +5,7 @@
 set -eux
 set -o pipefail
 
+UPDATECTL=/bin/updatectl
```

Comment:
IMO you don't need this. Just call `updatectl` directly. We do it for sysupdate to avoid typing out `/usr/lib/systemd/systemd-sysupdate` every time. But for updatectl, it's a user-facing command that should be in PATH. Let's make sure it is.